### DISCUSSION TITLE
Please upgrade your device's operating system and update to the newest version of Snapchat.

### DISCUSSION THREAD

**Discussion by ghost** [rodit/SnapMod#284] [General]
Prepare your anus's snapchat killing any apk under v12

**Comment by us3rname101** [comment]
Mans got a mappings file for the newest version?

**Reply by RevealedSoulEven** [reply]
Let rodit fix them

**Reply by us3rname101** [reply]
Aren't you the person to say let. You're literally begging him to update 💀

**Comment by us3rname101** [comment]
Fine. I'll do it myself. Couple of errors came up when creating them. 
![kuva](https://user-images.githubusercontent.com/95178906/202911602-6bd8a26b-4b51-4d5e-be09-be13b55e0aec.png)
The Snapmod button when long pressing users is gone and everything that modal had is probably gone.
[84658.txt](https://github.com/rodit/SnapMod/files/10050090/84658.txt)

**Reply by us3rname101** [reply]
lol forgor what snap version they are for, 12.08.0.29

**Comment by RevealedSoulEven** [comment]
Bro. I tried and I succeed too but.... Many features won't work

**Reply by us3rname101** [reply]
Sending videos seem to work now. I think that was an older bug?

**Reply by RevealedSoulEven** [reply]
With snapmod?

**Reply by us3rname101** [reply]
Yop

**Reply by Lucky-069** [reply]
Is the snapmod menu available in the settings and which features actually broke?

**Reply by RevealedSoulEven** [reply]
I didn't made mod for new version

**Reply by Lucky-069** [reply]
Could you try making it and see what works?

**Reply by RevealedSoulEven** [reply]
already in previous 5-6 versions I tried, it needs the codes to be fixed. And I can't do that

**Comment by No0099** [comment]


**Reply by Lucky-069** [reply]
It would have been great if you could just add the feature like @revealedsouleven did. The antisave button!

**Reply by RevealedSoulEven** [reply]
Use that bindings with my snapmod. He didn't modified the snapmod apk, only the bindings.

**Reply by Lucky-069** [reply]
> Use that bindings with my snapmod. He didn't modified the snapmod apk, only the bindings.

Oh okay I'll try

**Reply by melbarouzi** [reply]
What happend , is there any reason that makes you delete your Binding ??

**Reply by RevealedSoulEven** [reply]
He is doing the same since last month, everytime he changes account and keeps deleting his account. And this time he deleted the post but not the account, strange. Notice that it's a new account

**Reply by Lucky-069** [reply]
Who are you referring to?

**Reply by No0099** [reply]
> He is doing the same since last month, everytime he changes account and keeps deleting his account. And this time he deleted the post but not the account, strange. Notice that it's a new account

First, I've been waiting for 2 days for a response to my query on your Telegram... regarding some issues in Android Studio that I have no experience with... and so far the last reply I've got from you (Will see later)

Secondly, I only edit with my phone
Thirdly, how do you say that I changed accounts a month ago..?
Finally, I deleted the post for the purpose of fixing and will bring it back

**Reply by authorisation** [reply]
> > He is doing the same since last month, everytime he changes account and keeps deleting his account. And this time he deleted the post but not the account, strange. Notice that it's a new account
> 
> First, I've been waiting for 2 days for a response to my query on your Telegram... regarding some issues in Android Studio that I have no experience with... and so far the last reply I've got from you (Will see later)
> 
> Secondly, I only edit with my phone Thirdly, how do you say that I changed accounts a month ago..? Finally, I deleted the post for the purpose of fixing and will bring it back

Contact me I might be able to help you. t.me/auth1337

**Reply by RevealedSoulEven** [reply]
> > He is doing the same since last month, everytime he changes account and keeps deleting his account. And this time he deleted the post but not the account, strange. Notice that it's a new account
> 
> First, I've been waiting for 2 days for a response to my query on your Telegram... regarding some issues in Android Studio that I have no experience with... and so far the last reply I've got from you (Will see later)
> 
> Secondly, I only edit with my phone Thirdly, how do you say that I changed accounts a month ago..? Finally, I deleted the post for the purpose of fixing and will bring it back

Ahhh that guy is you?

**Reply by RevealedSoulEven** [reply]
> > He is doing the same since last month, everytime he changes account and keeps deleting his account. And this time he deleted the post but not the account, strange. Notice that it's a new account
> 
> First, I've been waiting for 2 days for a response to my query on your Telegram... regarding some issues in Android Studio that I have no experience with... and so far the last reply I've got from you (Will see later)
> 
> Secondly, I only edit with my phone Thirdly, how do you say that I changed accounts a month ago..? Finally, I deleted the post for the purpose of fixing and will bring it back

Btw you're great in finding those new classes.

**Reply by melbarouzi** [reply]
> > He is doing the same since last month, everytime he changes account and keeps deleting his account. And this time he deleted the post but not the account, strange. Notice that it's a new account
> 
> First, I've been waiting for 2 days for a response to my query on your Telegram... regarding some issues in Android Studio that I have no experience with... and so far the last reply I've got from you (Will see later)
> 
> Secondly, I only edit with my phone Thirdly, how do you say that I changed accounts a month ago..? Finally, I deleted the post for the purpose of fixing and will bring it back

Your binding is working well for must feature  with @RevealedSoulEven apk file

**Reply by RevealedSoulEven** [reply]
Hmmmm

**Reply by melbarouzi** [reply]
> Hmmmm
Yeap It works For Me Using your latest moded snapmod apk but i Didnt get this Hmmmm Meaning  ... Cause It makes me not comfortable  Using that binding file .

**Reply by Lucky-069** [reply]
> > Hmmmm
> > Yeap It works For Me Using your latest moded snapmod apk but i Didnt get this Hmmmm Meaning  ... Cause It makes me not comfortable  Using that binding file .

Could you share his bindings please. I accidentally deleted them

**Comment by RevealedSoulEven** [comment]
![Screenshot_20221202-000634_Gmail](https://user-images.githubusercontent.com/49313374/205133191-df589216-7287-4608-8b5a-033434546f14.png)

![Screenshot_20221202-000611_Gmail](https://user-images.githubusercontent.com/49313374/205133205-4e9dd420-28f6-44f5-9965-54948e484430.png)
If I ain't wrong, these are the same things happening and the posts get deleted. Maybe it's not you but activities were same btw.

**Reply by No0099** [reply]
> ![Screenshot_20221202-000634_Gmail](https://user-images.githubusercontent.com/49313374/205133191-df589216-7287-4608-8b5a-033434546f14.png)
> 
> ![Screenshot_20221202-000611_Gmail](https://user-images.githubusercontent.com/49313374/205133205-4e9dd420-28f6-44f5-9965-54948e484430.png) If I ain't wrong, these are the same things happening and the posts get deleted. Maybe it's not you but activities were same btw.

I don't know why you insist on this and I don't know what you want to get at but if it makes you feel comfortable I'll tell you it's me🙂🙄

**Reply by Lucky-069** [reply]
> > ![Screenshot_20221202-000634_Gmail](https://user-images.githubusercontent.com/49313374/205133191-df589216-7287-4608-8b5a-033434546f14.png)
> > ![Screenshot_20221202-000611_Gmail](https://user-images.githubusercontent.com/49313374/205133205-4e9dd420-28f6-44f5-9965-54948e484430.png) If I ain't wrong, these are the same things happening and the posts get deleted. Maybe it's not you but activities were same btw.
> 
> I don't know why you insist on this and I don't know what you want to get at but if it makes you feel comfortable I'll tell you it's me🙂🙄

Hey man, could you share your binding again. I actually lost them.. They worked great!! Now I'm stuck

**Reply by RevealedSoulEven** [reply]
Leave it bro.... It doesn't matters eho they are, can you please message me on telegram? https://t.me/soulevenofficial

@No0099
